lib/tarantool_server.py

```diff
@@ -816,6 +816,8 @@ def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
         if self.rpl_master:
             os.putenv("MASTER", self.rpl_master.iproto.uri)
         self.logfile_pos = self.logfile
+        self.signal_default = signal.SIGTERM
+        self.signal_kill = signal.SIGKILL
```

Comment:
I don't see a reason to omitting direct usage of `signal.SIGTERM` and `signal.SIGKILL`.